### DISCUSSION TITLE
Issues using {single-glossary-jlpt} with the Anki Integration

### DISCUSSION THREAD

**Discussion by LuisJavier23** [yomidevs/yomitan#1673] [Q&A]
Ok so first of all thank you a lot for all the work you guys do, I was kind of scared when the guy running yomichan said the app wasn't going to keep getting suppport given than the tool change a lot of how I learn Japanese and the fact that a lot of us can continue to keep learning japanese in the way we currently do it's thanks to all your hard work, so thank you, for real.

Anyways I have a little issue but I may be the one doing something wrong (or it could be an Anki issue, I wasn't sure where to ask first) 
I'm trying to use this frec dict as a field for an Anki card (so I can re sort them later) and the field it's totally there and the dropdown menu game me that option so in a way it thinks it's there, but when I export I only get an empty html div, any ideas of what could be happening here?

![image](https://github.com/user-attachments/assets/724dd78a-dccf-449f-959d-abf2e08b4244)
![image](https://github.com/user-attachments/assets/c684f036-5efe-401b-981e-59c6330237bd)

**Comment by Kuuuube** [comment]
Single dictionary handlebars for frequency and other types have not been added yet. But you can no longer select non term dictionaries for single glossary to avoid unexpected behavior.
